docs/packaging_policy.md

```diff
@@ -130,3 +130,14 @@ in this direction. However, there are a couple of options that have a special me
    ```
 
    ensuring that, when the option is active, the recipe ignores all the settings and only one package ID is generated.
+
+* `build_testing` should not be added, nor any other related unit test option. Options affect the package ID, therefore, testing should not be part of that.
+   Instead, use Conan config [skip_test](https://docs.conan.io/en/latest/reference/config_files/global_conf.html#tools-configurations) feature:
+
+   ```python
+   def _configure_cmake(self):
+      cmake = CMake(self)
+      cmake.definitions['BUILD_TESTING'] = self.conf.get("tools.build:skip_test", default=true, check_type=bool)
```

Comment:
```suggestion
      cmake.definitions['BUILD_TESTING'] = not self.conf.get("tools.build:skip_test", default=true, check_type=bool)
```
because we only want to build tests when skip_test is False